Frameworks/UIKit/XamlControls.mm

```diff
@@ -27,7 +27,9 @@
 // Button
 ////////////////////////////////////////////////////////////////////////////////////
 WXCButton* CreateButton() {
-    ComPtr<IInspectable> inspectable(XamlCreateButton());
+    ComPtr<IInspectable> inspectable;
+    // Use Attach for transfer-ownership semantics, else we +1 and leak
+    inspectable.Attach(XamlCreateButton());
```

Comment:
I suppose an out param would make it a bit more difficult to leak.

---
In reply to: [98310272](https://github.com/Microsoft/WinObjC/pull/1863#discussion_r98310272) [](ancestors = 98310272)